docs/extensibility/resources.md

```diff
@@ -176,6 +184,59 @@ renders the same set of data as:
 ]
 ```
 
+## relations section
+
+This section contains an object that maps a relation name to relation configuration object. Relation name preceded by dollar sign '\$' can be then used in `path` expression.
+
+It's possible to use both relation name and a path, e.g. `{"path": $myRelatedResource.metadata.labels}` will return the `metadata.labels` of related resource.
+
+### Relation configuration object fields
+
+Those fields are used to build the related resource URL and filter the received data.
+
+- **kind** - _[required]_ Kubernetes resource kind.
+- **group** - _[required]_ Kubernetes resource group.
+- **version** - _[required]_ Kubernetes resource version.
+- **namespace** - resource's Namespace name, defaults to original resource's Namespace. If set to `null` the relation with match clusterwide resources / resources in all namespaces.
```

Comment:
```suggestion
- **namespace** - the resource's Namespace name; it defaults to the original resource's Namespace. If set to `null`, the relation matches cluster-wide resources or resources in all Namespaces.
```

I'm guessing "with" is a typo for "will", and I'd say we can replace future tense with present tense here.